Assets/HoloToolkit/SpatialMapping/Scripts/SpatialMappingManager.cs

```diff
@@ -320,4 +320,4 @@ private void UpdateRendering(bool Enable)
             }
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Minor formtatting nitpick, but we've tended to leave out a final newline character in this repo.